docs: publish YDB CLI 2.16.0 with release notes (#12108)
ydb-platform/ydb: ydb/docs/en/core/changelog-cli.md

# {{ ydb-short-name }} CLI changelog

## Version 2.16.0 {#2-16-0}

Released on November 26, 2024. To update to version **2.16.0**, select the [Downloads](downloads/index.md#ydb-cli) section.

### Features

* Improved throughput of the `ydb import file csv` up to 3 times
* Allow running stock benchmark for OLAP shards
* Added support for ISO-8601 formatted timestamps in `ydb topic` commands
* Added `--explain-ast` option to `ydb sql` command that prints query AST
* Added ANSI SQL syntax highlighting in `ydb` interactive mode
* Added support for PostgreSQL syntax in `ydb workload tpch` и `ydb workload tpcds` benchmarks
* Added `-c` option for `ydb workload tpcds run` command to compare the result with expected value and show the diff
* In the `ydb topic write` command the `--codec` option now has default value RAW
* Added log events for `ydb tools dump` and `ydb tools restore` commands
* The `ydb tools restore` command now displays error locations

### Bug fixes

> * Fixed progress bar in `ydb workload import` command
> * Fixed a bug where restoring from a backup using --import-data could fail if the partitioning of the table was changed

Reviewer: ```suggestion
* Fixed the progress bar in the `{{ ydb-cli }} workload import` command.
* Resolved an issue where restoring from a backup using the `--import-data` option could fail if the table's partitioning had changed.
```
